assets/app/components/siteList/publishedState.js

```diff
@@ -12,7 +12,7 @@ const getLastBuildTime = (builds) => {
       let bCompletedAt = new Date(b.completedAt);
       return aCompletedAt > bCompletedAt;
     });
-    let last = sorted.pop();
+    let last = sorted.slice().pop();
```

Comment:
This has changed the functionality but the PublishedState test has not been changed. This seems to indicate our test for PublishedState wasn't as thorough as we wanted-- it looks like our test happened to only test if the last build was represented in the render and this hid the bug you found. Can we please add another test to exercise this new functionality to verify we don't happen to run into it again and further document the code with tests?